### DISCUSSION TITLE
Deep Link does not contain valid required params.

### DISCUSSION THREAD

**Discussion by kpoliveira** [invertase/react-native-firebase#6269] [General]
My custom build link it´s well formed and encoded, but, when i tap the link, the app opens but the xcode output shows the following error:

`[Firebase/Analytics][I-ACS023001] Deep Link does not contain valid required params. URL params: {
    link = "https://www.xyz.com/item/{myId}";
}`

**Comment by mikehardy** [comment]
With no other information to go on, I offer this https://firebase.google.com/docs/dynamic-links/debug
